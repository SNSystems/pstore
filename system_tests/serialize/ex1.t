# %binaries = the directories containing the executable binaries
# %T = the test output directory
# %S = the test source directory
REQUIRES: examples
RUN: %binaries/example_write_integers > %T/ex1_actual.txt
RUN: diff %T/ex1_actual.txt %S/ex1_expected.txt
