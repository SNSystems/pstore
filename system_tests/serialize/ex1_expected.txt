Wrote these bytes:
0a 00 00 00 14 00 00 00
